src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &_minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    width: $minutes-width;
+    height: $minutes-height;
+    transform: $hands-transform;
+    background-color: #0700ff;
+    z-index: 1;
+    transform-origin: 50% 100%;
+    animation: rotate 3600s infinite steps(60);
+  }
+
+  &_seconds-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    width: $seconds-width;
+    height: $seconds-height;
+    background-color: #2c8000;
+    z-index: 2;
+    transform: $hands-transform;
+    transform-origin: 50% 100%;
+    animation: rotate 60s infinite linear;
+  }
+
+  &_center-circle {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 3;
+  }
```

Comment:
The class name `&_center-circle` does not follow the BEM naming convention. It's recommended to use a double underscore `__` to separate the block from the element. The correct class name should be `&_center__circle`.